### PR TITLE
docs: update README to reflect deprecation

### PR DESCRIPTION
**Pull request by CaerusKaru**
Faretheewell, Flex Layout.

### CODE REVIEW

**Comment by CaerusKaru** [comment]
cc @alan-agius4 @mgechev @etwerkse

**Review by alan-agius4 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 [![Build status](https://circleci.com/gh/angular/flex-layout.svg?style=svg)](https://circleci.com/gh/angular/flex-layout)
 [![Gitter](https://badges.gitter.im/angular/flex-layout.svg)](https://gitter.im/angular/flex-layout)
 
+**NOTE: This project has been deprecated. Please review [this blog post](https://blog.angular.io/modern-css-in-angular-layouts-4a259dca9127) 
+for alternatives, and [this article](https://medium.com/@caerus.karu/farewell-flex-layout-aaa567023769) for the exaplanation and next steps.**
```
Comment (twerske) [inline README.md:8]:
```suggestion
for alternatives, and [this article](https://medium.com/@caerus.karu/farewell-flex-layout-aaa567023769) for the explanation and next steps.**
```

Comment (twerske) [reply README.md:8]: nit: spelling

**Review by twerske — APPROVED** [review]
Thank you @CaerusKaru! Left one spelling fix, otherwise LGTM

**Review by alan-agius4 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -4,6 +4,9 @@
 [![Build status](https://circleci.com/gh/angular/flex-layout.svg?style=svg)](https://circleci.com/gh/angular/flex-layout)
 [![Gitter](https://badges.gitter.im/angular/flex-layout.svg)](https://gitter.im/angular/flex-layout)
 
+**NOTE: This project has been deprecated. Please review [this blog post](https://blog.angular.io/modern-css-in-angular-layouts-4a259dca9127)
```
Comment (mgechev) [inline README.md:7]: Not sure deprecated is the correct phrasing. Maybe:

```suggestion
**NOTE: The Angular team no longer publishes new releases of this project. Please review [this blog post](https://blog.angular.io/modern-css-in-angular-layouts-4a259dca9127) 
```

**Pull request closed without merging**